lib/logstash/util/compress.rb

```diff
@@ -0,0 +1,116 @@
+# encoding: utf-8
+require "logstash/namespace"
+require "logstash/environment"
+
+module LogStash::Util
+  module Zip
+
+    extend self
+
+    def extract(source, target)
+      require "zip"
+      require "fileutils"
```

Comment:
any reason why lazy require? in general this should be avoided. please bump these at the top.